frontend/src/metabase/dashboard/components/Dashboard/Dashboard.tsx

```diff
@@ -484,21 +484,25 @@ function DashboardInner(props: DashboardProps) {
     >
       {() => (
         <DashboardStyled>
-          {isHeaderVisible && (
-            <DashboardHeaderContainer
-              isFullscreen={isFullscreen}
-              isNightMode={shouldRenderAsNightMode}
-            >
-              <DashboardHeader
-                {...props}
-                onEditingChange={handleSetEditing}
-                setDashboardAttribute={handleSetDashboardAttribute}
-                addParameter={addParameter}
-                parametersWidget={parametersWidget}
-                onSharingClick={handleToggleSharing}
-              />
-            </DashboardHeaderContainer>
-          )}
+          <DashboardHeaderContainer
+            isVisible={isHeaderVisible}
+            isFullscreen={isFullscreen}
+            isNightMode={shouldRenderAsNightMode}
+          >
+            {/**
```

Comment:
Please read the [PR description](https://github.com/metabase/metabase/pull/39005#issue-2146824908) for full details.